src/main/webapp/js/instructor.js

```diff
@@ -136,19 +135,6 @@ function setupFsCopyModal() {
 /**
  * @param elements:
  * identifier that points to elements with
- * class: profile-pic-icon-click or profile-pic-icon-hover
- */
-function bindErrorImages(elements) {
```

Comment:
This function is called in a few other places, and renaming it in `common.js` is what causes the Travis build to stall.